### ISSUE TITLE
Remove Modifier extensions that accept strings as inputs

### ISSUE THREAD

**Issue by bitspittle** [varabyte/kobweb#135]
This isn't type safe and is lazy. If people want those, they can use the `styleModifier` escape hatch.

This is a backwards incompatible change in a few cases so should be done before 1.0

**Comment by bitspittle** [comment]
Keep in mind we will probably want to support variables somehow. We can possibly introduce a CssVariable version for some methods, instead of taking string values directly, though.

**Comment by bitspittle** [comment]
Duplicate of #173
